azure-pipelines.yml

```diff
@@ -27,6 +27,8 @@ variables:
     value: $(ARTIFACTORY_URL)/api/npm/npm
   - name: isMain
     value: $[eq(variables['Build.SourceBranch'], 'refs/heads/master')]
+  - name: azureBranch
+    value: $[coalesce(variables['System.PullRequest.SourceBranch'], 'master')]
```

Comment:
Turns out we must use a pre-defined variable, otherwise it doesn't work if we directly put it in the `env` section of a task